dependencies/sweetalert/LICENSE

```diff
@@ -1,22 +0,0 @@
-The MIT License (MIT)
```

Comment:
I think we have to keep this copyright note in

Reply:
Resolved - it's in the node_modules repo